doc/man3/ERR_load_crypto_strings.pod

```diff
@@ -24,31 +24,27 @@ Deprecated:
 
 =head1 DESCRIPTION
 
-All of the following functions are deprecated from OpenSSL 1.1.0. No explicit
-initialisation or de-initialisation is necessary. See L<OPENSSL_init_crypto(3)>
-and L<OPENSSL_init_ssl(3)>.
-
 ERR_load_crypto_strings() registers the error strings for all
 B<libcrypto> functions. SSL_load_error_strings() does the same,
 but also registers the B<libssl> error strings.
 
-In versions of OpenSSL prior to 1.1.0 ERR_free_strings() freed all previously
-loaded error strings. However from OpenSSL 1.1.0 it does nothing.
+In versions prior to OpenSSL 1.1.0,
+ERR_free_strings() releases any resources created by the above functions.
 
 =head1 RETURN VALUES
 
 ERR_load_crypto_strings(), SSL_load_error_strings() and
 ERR_free_strings() return no values.
 
-=head1 SEE ALSO
-
-L<ERR_error_string(3)>
-
 =head1 HISTORY
 
 The ERR_load_crypto_strings(), SSL_load_error_strings(), and
 ERR_free_strings() functions were deprecated in OpenSSL 1.1.0 by
-OPENSSL_init_crypto() and OPENSSL_init_ssl().
+OPENSSL_init_crypto() and OPENSSL_init_ssl() and should not be used.
+
+=head1 SEE ALSO
+
+L<ERR_error_string(3)>
```

Comment:
Hmmm...  we usually have HISTORY after SEE ALSO, why are you changing that pattern in these pages?